### DISCUSSION TITLE
embedding in VueJS?

### DISCUSSION THREAD

**Discussion by ctSkennerton** [GMOD/jbrowse-components#1421] [Q&A]
I currently have a web app in VueJS, is it possible to embed jbrowse2 given that it's written with react?

**Comment by garrettjstevens** [comment]
The full JBrowse app isn't really designed to be embedded in another page. I don't have experience with VueJS development, but you might be able to add the JBrowse app to your public/static folder and link to it as a separate page. Or you could possibly use an `iframe`.

We are, however, developing a JBrowse Linear Genome View React component. This is the linear view (standard genome browser view) from JBrowse, and it is designed to be embedded inside any React application. I think with that you could use something like [vuera](https://github.com/akxcv/vuera) and put it inside your Vue application. Stay tuned for more information on the forthcoming React component.
